src/api/worker/crypto/CryptoFacade.ts

```diff
@@ -38,36 +38,38 @@ import {
 	PermissionTypeRef,
 	PushIdentifierTypeRef,
 } from "../../entities/sys/TypeRefs.js"
-import type { Contact, InternalRecipientKeyData, Mail, SymEncInternalRecipientKeyData } from "../../entities/tutanota/TypeRefs.js"
 import {
+	Contact,
 	ContactTypeRef,
 	createEncryptTutanotaPropertiesData,
 	createInternalRecipientKeyData,
 	createSymEncInternalRecipientKeyData,
 	File,
 	FileTypeRef,
+	InternalRecipientKeyData,
+	Mail,
 	MailTypeRef,
+	SymEncInternalRecipientKeyData,
 	TutanotaPropertiesTypeRef,
 } from "../../entities/tutanota/TypeRefs.js"
 import { typeRefToPath } from "../rest/EntityRestClient"
 import { LockedError, NotFoundError, PayloadTooLargeError, TooManyRequestsError } from "../../common/error/RestError"
-import { SessionKeyNotFoundError } from "../../common/error/SessionKeyNotFoundError" // importing with {} from CJS modules is not supported for dist-builds currently (must be a systemjs builder bug) // importing with {} from CJS modules is not supported for dist-builds currently (must be a systemjs builder bug)
+import { SessionKeyNotFoundError } from "../../common/error/SessionKeyNotFoundError" // importing with {} from CJS modules is not supported for dist-builds currently (must be a systemjs builder bug) // importing with {} from CJS modules is not supported for dist-builds currently (must be a systemjs builder bug) // importing with {} from CJS modules is not supported for dist-builds currently (must be a systemjs builder bug)
```

Comment:
did this comment multiply?
It anyway does not belong here it seems

Reply:
I don't know where it belongs, so it is just going away.